source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1544,14 +1544,16 @@ export abstract class BaseSubmitActionPeer<TAction extends Adaptive.SubmitAction
             { targetVersion: Adaptive.Versions.v1_3, name: "Automatic", value: "auto" },
             { targetVersion: Adaptive.Versions.v1_3, name: "None", value: "none" }
         ]);
+    static readonly disabledUnlessAssociatedInputsChangeProperty = new BooleanPropertyEditor(Adaptive.Versions.v1_0, "disabledUnlessAssociatedInputsChange", "Only enable when associated inputs change");
```

Comment:
`Only enable when associated inputs change` is too long

Reply:
@licanhua What other name do you suggest?

Reply:
remove it? and associatedinputs != none means disabledUnlessAssociatedInputsChange

Reply:
That's completely different. `associatedInputs` controls which inputs are associated with the action. The new property controls if the action should be disabled if none of its associated inputs have changed.

Reply:
I can only think of 

EnableAssociatedAction or DisableAssociatedAction

Reply:
@JeanRoca doesn't sound too good to me. This property lives on Action. This name would make it sound like the action has an associated action.